<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartPage</class>
 <widget class="QWidget" name="StartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1601</width>
     <height>1201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid">
    <item row="3" column="4">
     <spacer name="hspace_r">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" rowspan="5">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="button_accept">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="h_title">
      <item>
       <spacer name="hspace_l_title">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="v_title">
        <item>
         <spacer name="vspace_t_title">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="text_title_title">
          <property name="font">
           <font>
            <pointsize>26</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>S-Alert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="text_login_title">
          <property name="text">
           <string>Create Account</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="vspace_b_title">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="hspace_r_title">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="6" column="2">
     <spacer name="vspace_b">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>100</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <spacer name="hspace_l">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="vspace_t">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>100</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLineEdit" name="textbox_enter"/>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLabel" name="text_account">
      <property name="text">
       <string>Enter an account name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLabel" name="text_create">
      <property name="text">
       <string>You must create a Level 1 account</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="button_back">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
